src/store/ChartStore.js

```diff
@@ -519,6 +519,14 @@ class ChartStore {
             this.stxx = null;
         }
     }
+
+    @action.bound onHistorical() {
+        this.contextPromise.then(() => {
+            // console.log('asdas');
+            this.mainStore.chartType.setType('mountain');
+            this.changeSymbol(undefined, 0);
```

Comment:
So I understand you added `onHistorical` so that when the user selects historical mode you change to a line chart with ticks granularity. 

A better way to do this is from the app. When the app gets the settings that it is in historical mode, it sets these exact same settings using props